apollo-router/src/plugins/connectors/testdata/interface-object.yaml

```
@@ -0,0 +1,48 @@
+# rover supergraph compose --config apollo-router/src/plugins/connectors/testdata/interface-object.yaml > apollo-router/src/plugins/connectors/testdata/interface-object.graphql
+# the result is manually edited because 2.10 disallows interface object
+federation_version: =2.10.0-alpha.1
+subgraphs:
+  connectors:
+    routing_url: none
+    schema:
+      sdl: |
+        extend schema
+          @link(url: "https://specs.apollo.dev/federation/v2.10", import: ["@key"])
+          @link(
+            url: "https://specs.apollo.dev/connect/v0.1"
+            import: ["@connect", "@source"]
+          )
+          @source(name: "json", http: { baseURL: "http://localhost:4001" })
+
+        type Query {
+          x: [Itf2]
+            @connect(
+              source: "json"
+              http: { GET: "/itf" }
+              selection: "id"
+            )
+        }
+
+        type Itf2 @key(fields: "id") {
```

```suggestion
        type Itf2 @key(fields: "id") @interfaceObject {
```

Is this how this works? I haven't used it since it first came out 😓

---

yeah — what i did was compose without the interfaceObject (and using `Itf` vs `Itf2`) and then manually editing the supergraph. i can add this behind a comment.